tasks/main.yml

```diff
@@ -1,13 +1,15 @@
 ---
+- name: Set default variables
+  include_vars: default.yml
```

Comment:
this should not be needed, if you need to set a variable for all the distributions, set it in vars/main.yml. default.yml is intended for unrecognized distributions.